queries/dart/highlights.scm

```diff
@@ -109,37 +139,61 @@
 
 ; Keywords
 
+(this) @keyword
+
 ["import" "library" "export"] @include
 
+; TODO: "rethrow" @keyword
+
+; Reserved words (cannot be used as identifiers)
 [
- "abstract"
- ;"assert"
- ;"break"
- "on"
- "class"
- "default"
- "enum"
- "extends"
- ;"final"
- "implements"
- "as"
- "mixin"
- "external"
- "new"
- "return"
- "static"
- "required"
- "var"
- ;"const"
- "async"
- "await"
- ] @keyword
-;TODO:
-; "rethrow" @keyword
+    ; "assert"
+    ; "break"
+    "class"
+    "enum"
+    "extends"
+    "in"
+    "is"
+    "new"
+    "return"
+    "super"
+    "var"
+    "with"
+    ; "void"
+] @keyword
+
+[
+    "abstract"
+    "as"
+    "async"
+    "await"
+    "covariant"
+    "deferred"
+    "dynamic"
+    "external"
+    "factory"
+    "Function"
+    "get"
+    "implements"
+    "interface"
+    "library"
+    "operator"
+    "mixin"
+    "part"
+    "set"
+    "static"
+    "typedef"
+] @keyword
 
 ["if" "else" "switch" "default"] @conditional
-; TODO: case
 
 ["try" "throw" "catch" "finally"] @exception
 
 ["do" "while" "continue" "for"] @repeat
+
+((identifier) @variable.builtin
```

Comment:
This is a wholesale copy-pasta from the upstream `highlights.scm` I don't think it currently works and tbh I don't really know what it's supposed to do 😅

Reply:
#is-not is not working yet for highlights. See #114 

There matches if it matches the regex for a keyword, but is not defined as a local variable. Apparently you can define a variable as in dart and it will overwrite the key word. Just drop `#is-not`